<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_pub</class>
 <widget class="QWidget" name="Edit_pub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ep_name_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название издательства</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ep_name_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ep_gen_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Генеральный директор</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ep_gen_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ep_address_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Адрес</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ep_address_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ep_cancel_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ep_delete_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ep_add_button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ep_save_button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
